packages/larva/lib/writeJson.js

```diff
@@ -41,7 +41,11 @@ module.exports = function writeJson( patternConfig, fromLarva = false ) {
 
 			if ( hasChanged ) {
 				writeJsonToFile( jsonDestPath, moduleData );
-				console.log( chalk.green.bold( `Wrote JSON for ${moduleName}.${variant}` ) );
+				if ( moduleData.error ) {
```

Comment:
Thank you for this contribution! One suggestion - what about using a try / catch block and throwing an exception for the errors? If there are many modules, the output of this command is quite long and I think the notice of errors could be lost if they are early in the output. My inclination is that this kind of error is serious enough to merit exiting the process. What do you think?

Reply:
Also, please add a test case here: https://github.com/penske-media-corp/pmc-larva/blob/master/packages/larva/__tests__/lib/writeJson.test.js

Reply:
Hi, @laras126, sorry for my delayed response. I plan to follow your suggestions and get this completed this weekend. Thanks!